src/symbol/constants.rs

```diff
@@ -576,6 +595,12 @@ impl From<u8> for SourceLanguage {
             0x0e => Self::JScript,
             0x0f => Self::MSIL,
             0x10 => Self::HLSL,
+            0x11 => Self::ObjC,
+            0x12 => Self::ObjCXX,
+            0x13 => Self::Swift,
+            0x14 => Self::AliasObj,
+            0x16 => Self::Rust,
+            0x17 => Self::Go,
```

Comment:
Typos? These should be `0x15` and `0x16`.

Reply:
Yes, definitely. Otherwise the values of the enum and the From<u8> impl don't match